api-src/org/labkey/api/targetedms/model/SampleFileInfo.java

```diff
@@ -51,6 +52,16 @@ public void setSampleFile(String sampleFile)
         this.sampleFile = sampleFile;
     }
 
+    public int getSampleFileId()
```

Comment:
It looks like this got added but not used. See other comment about potentially using it instead of concatenating two other columns